README.md

```diff
@@ -0,0 +1,81 @@
+Dell PowerConnect Switch
+========================
+
+This role configures Dell PowerConnect switches using the `expect` Ansible
+module.
```

Comment:
Might be user-friendly to put a link to http://docs.ansible.com/ansible/latest/modules/expect_module.html?highlight=expect in here.

Reply:
Good idea.